File: go/raindrops/raindrops.go
package raindrops

import "strconv"

func Convert(n int) (result string) {
	if n%3 != 0 && n%5 != 0 && n%7 != 0 {
		result = strconv.Itoa(n)
	}

	if n%3 == 0 {
		result += "Pling"
	}

	if n%5 == 0 {
		result += "Plang"
	}

	if n%7 == 0 {
		result += "Plong"
	}

	return
}

File: cpp/raindrops/raindrops.cpp
#include "raindrops.h"
#include <string>

namespace raindrops {
  std::string convert(int n) {
    std::string s{""};
    if (n % 3 == 0) {
      s += "Pling";
    }
    if (n % 5 == 0) {
      s += "Plang";
    }
    if (n % 7 == 0) {
      s += "Plong";
    }

    if (s.length() == 0) {
      return std::to_string(n);
    }

    return s;
  }
} // namespace raindrops
